src/notebook/plugin.ts

```diff
@@ -68,6 +68,7 @@ const cmdIds = {
   restartRunAll: 'notebook:restart-runAll',
   switchKernel: 'notebook:switch-kernel',
   clearAllOutputs: 'notebook:clear-outputs',
+  killAndExit: 'notebook:halt-and-close',
```

Comment:
Perhaps `closeAndHalt` and `close-and-halt` and `Close and Halt` to be consistent?